vendor/vendor.json

```diff
@@ -718,7 +718,7 @@
 			"revisionTime": "2017-05-10T02:47:26Z"
 		},
 		{
-			"checksumSHA1": "oBgjrQKzT1cPwaRQy4QSgRCqSeo=",
```

Comment:
Not sure what happened here.. Did you change something or is this something strange in Windows vs Mac/Linux?